staging/src/k8s.io/client-go/util/workqueue/default_rate_limiters.go

```diff
@@ -62,6 +62,49 @@ func (r *BucketRateLimiter) NumRequeues(item interface{}) int {
 func (r *BucketRateLimiter) Forget(item interface{}) {
 }
 
+type ItemBucketRateLimiter struct {
+	r     rate.Limit
```

Comment:
instead of using rate.Limit, I would prefer using "qps" and "burst" names explicitly (for consistency with every else).

Reply:
As we discussed offline, I kept rate.Limit